server/utils/nunjucksSetup.ts

```diff
@@ -79,5 +81,109 @@ export default function nunjucksSetup(app: express.Application): nunjucks.Enviro
       checked: item === value,
     }))
   })
+
+  njkEnv.addFilter('toCourtTypeFilterCheckboxes', (courtTypes: CourtType[], allCourtsFilter: AllCourtsFilter) => {
+    const courtTypeItems = courtTypes.map((courtType: CourtType) => {
+      return {
+        value: courtType.courtType,
+        text: courtType.courtName,
+        checked: allCourtsFilter.courtTypeIds.includes(courtType.courtType),
+      }
+    })
+    return {
+      idPrefix: 'courtType',
+      name: 'courtType',
+      classes: 'govuk-checkboxes--small',
+      fieldset: {
+        legend: {
+          text: 'Court Types',
+          classes: 'govuk-fieldset__legend--m',
+        },
+      },
+      items: courtTypeItems,
+    }
+  })
+
+  njkEnv.addFilter('toActiveFilterRadioButtons', (allCourtsFilter: AllCourtsFilter) => {
```

Comment:
Generates GDS style radio butrtons for the active statuses All/Open/Closed based on the spec. here: https://design-system.service.gov.uk/components/radios/